proxy.py

```
@@ -1270,6 +1347,38 @@ async def anthropic_messages(request: Request, auth: AuthContext = Depends(verif
     )
 
 
+class _CountTokensRequest(BaseModel):
+    model: str = Field(..., min_length=1, max_length=200)
+    messages: list[dict] = Field(default_factory=list)
+    system: str | None = None
+    tools: list[dict] | None = None
+
+
+@app.post("/v1/messages/count_tokens")
+async def count_tokens(request: Request, auth: AuthContext = Depends(verify_api_key)):
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)